Options.py

```diff
@@ -1170,7 +1191,9 @@ def dictify_range(option: Range):
 
     for game_name, world in AutoWorldRegister.world_types.items():
         if not world.hidden or generate_hidden:
-            all_options: typing.Dict[str, AssembleOptions] = world.options_dataclass.type_hints
+            all_options: typing.Dict[str, AssembleOptions] = {
+                option_name: option for option_name, option in world.options_dataclass.type_hints.items()
+                if option.visibility & Visibility.template}
```

Comment:
```suggestion
                if option.visibility & Visibility.template
            }
```
There was an inconsistency in this PR (with the `WebHostLib` changes) with whether the closing bracket is on its own line (when the opening bracket is at the end of a line).
My preference is for it to be on a new line.
Either way is ok, but I think it's better if it's consistent.
(Maybe it would be a good addition to the style guide.)